src/main/java/com/auth0/client/auth/PasswordlessEmailType.java

```diff
@@ -0,0 +1,32 @@
+package com.auth0.client.auth;
+
+/**
+ * Represents the type of the Passwordless email request.
+ */
+public enum PasswordlessEmailType {
```

Comment:
How about `PasswordlessEmailConnection`, since it represents the value of the `connection` parameter for email ([docs](https://auth0.com/docs/api/authentication#get-code-or-link))?